website/src/components/downloads/OSDownloadCard.tsx

```
@@ -0,0 +1,181 @@
+
+  const toggleExpanded = (): void => {
+    setExpanded(value => !value);
+  };
+
+  return (
+    <div
+      className={`rounded-lg dark:text-gray-400 text-charcoal-300 bg-zinc-300/25 dark:bg-zinc-700/25 ${
+        highlighted ? 'ring-2 ring-purple-500' : ''
+      }`}>
+      <div
+        role="button"
```

Biome (2.5.8)

[error] The elements with this role can be changed to the following elements:
<button>

(lint/a11y/useSemanticElements)